mevolib/cluster/Genes.py

```diff
@@ -125,215 +85,171 @@
+    # sequence during the alignment process
     record_seq = ''.join((x  for i, x in enumerate(alignment[1])
                                  if alignment[0][i] != '-'))
-    return ( Seq(record_seq, refseq_record.seq.alphabet),
-             refseq_record.features )
+    return (Seq(record_seq, refseq_record.seq.alphabet),
+             refseq_record.features)
 
 
-
-def _string_filter ( input_str ) :
-    """
-    Arguments :
-        input_str  ( string )
-            Input string to filter.
-
-    Returns :
-        string
-            Lower case of 'input_str' with known GenBank's misspellings
-            corrected.
+def _string_filter(input_str: str) -> str:
+    """Returns a string lower case of 'input_str' with known GenBank's misspellings corrected.
+    Args:
+        input_str: Input string to filter.
     """
     # Ignore case
     final_str = input_str.lower()
     # Correct misspellings
     final_str = final_str.replace('oxydase', 'oxidase')
-    return ( final_str )
-
-
-#-------------------------------------------------------------------------------
+    return final_str
 
-def get_features ( ) :
-    """
-    Returns :
-        list
-            List of all possible feature keywords that can be found in any
-            GenBank's sequence record.
-    """
-    return ( [x  for x in iter(viewkeys(_FEAT_QUAL_DICT))] )
 
+def get_features() -> List:
+    """Returns a list of all possible feature keywords that can be found in any GenBank's sequence record."""
+    return [x for x in iter(_FEAT_QUAL_DICT.keys())]
 
 
-def map_seqs ( record_list, feature_filter = None, ref_seq = None,
-               alignment_bin = None, log_file = None ) :
-    """
-    Gene splicing of the sequences at 'record_list'. By default, the gene
-    location is extracted from the feature list of each sequence. If there is no
-    list, that sequence is classified as "unprocessable" or, if a reference
-    sequence is given, the reference features are used to extract the different
-    genes (through a normalization process using an alignment tool). All the
-    features are returned unless a list of feature keywords are passed through
-    'feature_filter' parameter. If a log file path is given and any file exists
+def map_seqs (record_list: list, feature_filter: Optional[List] = None, ref_seq: Optional[str] = None, 
+              alignment_bin: Optional[str] = None, log_file: Optional[str] = None) -> dict:
+    """Gene splicing of the sequences at 'record_list'. By default, the gene location is extracted from the 
+    feature list of each sequence. If there is no list, that sequence is classified as "unprocessable" or, if 
+    a reference sequence is given, the reference features are used to extract the different genes (through a 
+    normalization process using an alignment tool). All the features are returned unless a list of feature 
+    keywords are passed through 'feature_filter' parameter. If a log file path is given and any file exists
```

Comment:
_Note:_ You will need to remove the next line (GitHub does not allow me to touch it)
```suggestion
    """Gene splicing of the sequences at `record_list`.

    By default, the gene location is extracted from the feature list of each sequence. If there is
    no list, that sequence is classified as "unprocessable" or, if a reference sequence is given,
    the reference features are used to extract the different genes (through a normalization process
    using an alignment tool). All the features are returned unless a list of feature keywords are
    passed through `feature_filter`. If a log file path is given and any file exists with that name,
    the file will be overwritten without any warning.
```